### ISSUE TITLE
building successfully but no internet

### ISSUE THREAD

**Issue by bigmoveus** [hossinasaadi/sshTunnel-ios#5]
Hi , Can building successfully but no internet . server working and tested with other app working well ! possible check it ?

**Comment by hossinasaadi** [comment]
did you enable udpgw on server? like :
https://gist.github.com/1nikolas/4efbfb6b0d37eba8c9ab471c55f5e892

**Comment by bigmoveus** [comment]
Yes problem is not it . It works with other app but it does not works in your code mean it connected but has not İnternet

**Comment by hossinasaadi** [comment]
your server works on demo app i published on appstore?
https://apps.apple.com/us/app/ssh-vpn-client-for-ssh/id6471132702

**Comment by bigmoveus** [comment]
Yes it is working with that app but does not works with your code

**Comment by hossinasaadi** [comment]
you should debug badvpn so what is error
